llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -59,6 +59,13 @@ class IREELinalgExt_Op<string mnemonic, list<Trait> traits = []> :
 // Non-structured ops
 //===----------------------------------------------------------------------===//
 
+def OpGroupNonStructuredOps : OpDocGroup {
+  let summary = "Non-structured ops";
```

Comment:
Is this grouping accurate? Should the ops in this file be moved around?

![image](https://github.com/openxla/iree/assets/4010439/7e10a6dd-ea95-41b5-8c53-f5e121d90a2f)

Reply:
I feel that all of them are non-structured ops. (actually, I don't have a definition for non-structure ops. We have all these ops because they don't play well with Linalg ops and transforms.)

I'd suggest to add  a `Data Tiling ops` category, which contains `pack`, `unpack`, `set_encoding`, `unset_encoding` ops; put the rest of ops to `Misc Non-structured ops`. Something like:

```
Misc. Non-structured ops
  - fft
  - reverse
  - scan
  - scatter
  - sort
  - topk
  - softmax
 
Data Tiling ops
  - set_encoding
  - unset_encoding
  - pack
  - unpack
```

Reply:
Moved some ops and groups around the in file, now looks like this:
![image](https://github.com/openxla/iree/assets/4010439/10d7c798-aeac-4760-a73b-4444b0bccad9)